src/axom/core/CMakeLists.txt

```diff
@@ -99,10 +95,12 @@ set(core_sources
 #------------------------------------------------------------------------------
 
 set( core_depends fmt )
+blt_list_append( TO core_depends ELEMENTS adiak::adiak IF ADIAK_FOUND )
+blt_list_append( TO core_depends ELEMENTS caliper IF CALIPER_FOUND )
 blt_list_append( TO core_depends ELEMENTS camp IF CAMP_FOUND )
 blt_list_append( TO core_depends ELEMENTS umpire IF UMPIRE_FOUND )
 blt_list_append( TO core_depends ELEMENTS RAJA IF RAJA_FOUND )
-blt_list_append( TO core_depends ELEMENTS nvToolsExt IF AXOM_ENABLE_CUDA )
+blt_list_append( TO core_depends ELEMENTS nvToolsExt IF AXOM_ENABLE_CUDA ) ## TODO: Remove
```

Comment:
Are you sure we should remove this? It may provide additional insight into CUDA GPU execution that is not available in caliper.

Reply:
Thanks for noticing that one -- I added that to check if we need to explicitly add the `nvToolsExt` dependency to `axom::core` or if we now get it from the `caliper` target.  

There should have been a question mark at the end.

Reply:
I was able to remove the line while preserving `nvtx` annotations through caliper